src/utils/privileges.cpp

```diff
@@ -36,6 +53,7 @@ void drop_privileges_on_startup() {
 	change.set_given_cap("NET_RAW", {capmodpp::v_eff_unchanged, capmodpp::v_permit_unchanged, capmodpp::v_inherit_unchanged}); // not yet used
 	change.set_given_cap("NET_BIND_SERVICE", {capmodpp::v_eff_unchanged, capmodpp::v_permit_unchanged, capmodpp::v_inherit_unchanged}); // not yet used
 	change.security_apply_now();
+	change_user_if_root();
```

Comment:
here we should catch the above special exception class, and then print:
_warn("Can not drop privileges to a regular user. We suggest to instead from a regular user call our program with sudo, and not run it directly as root");
and continue.

The other exceptions representing real errors in using system functions, should be NOT catched and should propagate